"npm run lint" error

I found the following error, when executing `npm run lint` on the latest master, so I will try to fix those.
```
% npm run lint

> @siedlerchr/types-ol-ext@1.0.0 lint /Users/sanak/Projects/gtt-project/git/types-ol-ext-gtt
> tsc -p tsconfig.lint.json --pretty && tslint -p tsconfig.lint.json

@types/ol-ext/control/EditBar.d.ts:54:5 - error TS2416: Property 'getControls' in type 'EditBar' is not assignable to the same property in base type 'Bar'.
  Type '() => void' is not assignable to type '() => Control[]'.
    Type 'void' is not assignable to type 'Control[]'.

54     getControls(): void;
       ~~~~~~~~~~~

@types/ol-ext/control/IsochroneGeoportail.d.ts:16:19 - error TS2304: Cannot find name 'Feature'.

16     getTitle: (f: Feature) => string;
                     ~~~~~~~

@types/ol-ext/control/IsochroneGeoportail.d.ts:17:20 - error TS7051: Parameter has a name but no type. Did you mean 'arg0: string'?

17     autocomplete: (string, ...params: any[]) => any; // TODO: not sure about the syntax
                      ~~~~~~

@types/ol-ext/control/Target.d.ts:36:5 - error TS2416: Property 'getCanvas' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '(map: Map) => HTMLCanvasElement'.
    Type 'void' is not assignable to type 'HTMLCanvasElement'.

36     getCanvas(): void;
       ~~~~~~~~~

@types/ol-ext/control/Target.d.ts:44:5 - error TS2416: Property 'getStyle' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '() => Style'.
    Type 'void' is not assignable to type 'Style'.

44     getStyle(): void;
       ~~~~~~~~

@types/ol-ext/control/Target.d.ts:48:5 - error TS2416: Property 'getStroke' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '() => Stroke'.
    Type 'void' is not assignable to type 'Stroke'.

48     getStroke(): void;
       ~~~~~~~~~

@types/ol-ext/control/Target.d.ts:52:5 - error TS2416: Property 'getFill' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '() => Fill'.
    Type 'void' is not assignable to type 'Fill'.

52     getFill(): void;
       ~~~~~~~

@types/ol-ext/control/Target.d.ts:56:5 - error TS2416: Property 'getTextStroke' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '() => Stroke'.
    Type 'void' is not assignable to type 'Stroke'.

56     getTextStroke(): void;
       ~~~~~~~~~~~~~

@types/ol-ext/control/Target.d.ts:60:5 - error TS2416: Property 'getTextFill' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '() => Fill'.
    Type 'void' is not assignable to type 'Fill'.

60     getTextFill(): void;
       ~~~~~~~~~~~

@types/ol-ext/control/Target.d.ts:64:5 - error TS2416: Property 'getTextFont' in type 'Target' is not assignable to the same property in base type 'CanvasBase'.
  Type '() => void' is not assignable to type '() => string'.
    Type 'void' is not assignable to type 'string'.

64     getTextFont(): void;
       ~~~~~~~~~~~


Found 10 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @siedlerchr/types-ol-ext@1.0.0 lint: `tsc -p tsconfig.lint.json --pretty && tslint -p tsconfig.lint.json`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @siedlerchr/types-ol-ext@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sanak/.npm/_logs/2021-06-23T13_58_52_923Z-debug.log
```

--- a/@types/ol-ext/control/EditBar.d.ts
+++ b/@types/ol-ext/control/EditBar.d.ts
@@ -51,7 +51,7 @@ export default class EditBar extends Bar {
     /** Get controls in the panel
     *	@param {Array<_ol_control_>}
      */
-    getControls(): void;
+    getControls(): ol_control_Control[];
     /** Set tool bar position
     *	@param {top|left|bottom|right} pos
      */

--- a/@types/ol-ext/control/IsochroneGeoportail.d.ts
+++ b/@types/ol-ext/control/IsochroneGeoportail.d.ts
@@ -1,6 +1,7 @@
 import { Map as _ol_Map_ } from 'ol';
 import ol_control_Control from 'ol/control/Control';
 import { Coordinate } from 'ol/coordinate';
+import Feature from 'ol/Feature';
 
 export interface Options {
     className: string;
@@ -14,7 +15,7 @@ export interface Options {
     maxItems: number | undefined;
     maxHistory: number | undefined;
     getTitle: (f: Feature) => string;
-    autocomplete: (string, ...params: any[]) => any; // TODO: not sure about the syntax
+    autocomplete: (s: string, ...params: any[]) => any; // TODO: not sure about the syntax
     exclusions: string;
 }
 /**

--- a/@types/ol-ext/control/Target.d.ts
+++ b/@types/ol-ext/control/Target.d.ts
@@ -31,35 +31,4 @@ export default class Target extends CanvasBase {
      * @api stable
      */
     setMap(map: _ol_Map_): void;
-    /** Get canvas overlay
-     */
-    getCanvas(): void;
-    /** Set Style
-     * @api
-     */
-    setStyle(): void;
-    /** Get style
-     * @api
-     */
-    getStyle(): void;
-    /** Get stroke
-     * @api
-     */
-    getStroke(): void;
-    /** Get fill
-     * @api
-     */
-    getFill(): void;
-    /** Get stroke
-     * @api
-     */
-    getTextStroke(): void;
-    /** Get text fill
-     * @api
-     */
-    getTextFill(): void;
-    /** Get text font
-     * @api
-     */
-    getTextFont(): void;
 }

